src/main/java/org/jboss/pnc/causeway/brewclient/BuildTranslatorImpl.java

```diff
@@ -257,7 +279,8 @@ public String getSourcesDeployPath(org.jboss.pnc.dto.Build build, BuildArtifacts
                 artifacts,
                 (gav) -> renamer.getMavenDeployPath(gav.getGroupId(), gav.getArtifactId(), gav.getVersionString()),
                 (npmPackage) -> renamer
-                        .getNPMDeployPath(npmPackage.getName(), npmPackage.getVersion().getNormalVersion()));
+                        .getNPMDeployPath(npmPackage.getName(), npmPackage.getVersion().getNormalVersion()),
+                (gav) -> null);
```

Comment:
I think MVN_RPM should also be considered? Can't remember what this does but it seems important.

Reply:
I return null as that is used in ImportControllerImpl::getSources which then falls back to calling BuildTranslatorImpl::getSources which will also then return null - this is because there is/should be no source jar for RPMs - its instead handled by the src rpm which is generated by rpmbuild.